core/vm/actor/cgo/go_actors.go

```diff
@@ -172,7 +171,8 @@ func (rt *rt) Send(to address.Address, method abi.MethodNum, o cbor.Marshaler, v
 	return exit
 }
 
-func (rt *rt) Abortf(exit exitcode.ExitCode, _ string, _ ...interface{}) {
+func (rt *rt) Abortf(exit exitcode.ExitCode, msg string, args ...interface{}) {
+    fmt.Printf("Abort: " + msg, args...)
```

Comment:
```suggestion
	fmt.Printf("Abort:", msg, args...)
```

Reply:
No, msg contains format string (%s, etc), and "Abort: " is just a prefix to identify that execution was aborted.